sonar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "Building docker image"
+docker build -f ./config/sonar.dockerfile -t sonar ./config/
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonar:latest
+echo "sleeping 60 sec to let sonar start up"
+sleep "60"
+
+#set checkstyle sonar profile
+curl -X POST -u admin:admin -F 'backup=@config/default_sonar_profile.xml' -v http://localhost:9000/api/profiles/restore
+
+#execute inspection
+#do it 2 times to get nice html report and parsable json.
+mvn sonar:sonar
+mvn sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.issuesReport.html.enable=true
```

Comment:
Do we really need html format ?
it is not required for CI, we could put it to developers manual only when it comes to execute on local and review and probably share on github.io  .